epub33/core/index.html

```diff
@@ -657,15 +657,37 @@ <h4>Introduction</h4>
 						without fallbacks (<a>Core Media Type Resources</a>) and those that cannot (<a>Foreign
 							Resources</a>).</p>
 
-					<p><a>Authors</a> are free to use both types of resources to construct their EPUB Publications, but
-						need to be aware that some Reading Systems might not render the Foreign Resources they use.</p>
-
-					<p>As EPUB Publications are designed to be fully consumable on any compliant Reading System, a
-						system of fallbacks is necessary to ensure that the use of Foreign Resources does not impact on
-						the ability of the user to consume the content. This section lists the <a
-							href="#sec-cmt-supported">set of Core Media Type Resources</a> and identifies <a
-							href="#sec-foreign-restrictions">fallback mechanisms</a> that can be used to satisfy the
-						consumability requirement.</p>
+					<p>Formats are typically only included as Core Media Type Resources when it can be shown that they
+						have broad support in web browser cores &#8212; the rendering engines on which EPUB 3 Reading
+						Systems are built. They are like a compact between <a>Authors</a> and Reading Systems to ensure
+						the predictability of rendering of EPUB Publications.</p>
+
+					<p>Inclusion as a Core Media Type Resource does not mean that all Reading Systems will support the
+						rendering of a resource, however. Only those Reading Systems that support the type of rendering
+						related to the resource are expected to (e.g., a Reading System with a <a>Viewport</a> has to
+						support image Core Media Type Resources, but a Reading System without a Viewport does not).</p>
```

Comment:
What does "type of rendering related to the resource" mean? Where is that defined? (If there is a definition than a cross-reference would be helpful.)

Reply:
There's a reference to the relevant reading system requirements in the immediately following note, although it could be integrated as prose. I couldn't think of any better way of saying that support depends on what the reading system is capable of rendering. I'll have another look at whether it can be simplified.

Reply:
This para now read:

> Inclusion as a Core Media Type Resource does not mean that all Reading Systems will support the rendering of a resource, however. Only Reading Systems that are capable of rendering the type of resource have to (e.g., a Reading System with a Viewport has to support image Core Media Type Resources, but a Reading System without a Viewport does not). Refer to Conformance Requirements [EPUB-RS-33] for more information about which Reading Systems rendering capabilities require support for which Core Media Type Resources.

Reply:
The current cross references to RS go 404 (if they refer to /TR), but I trust you on this.